newIDE/app/src/MainFrame/ElectronMainMenu.js

```diff
@@ -400,5 +394,6 @@ class ElectronMainMenu extends React.Component<Props, {||}> {
     });
   }
 }
+ElectronMainMenu.contextType = PreferencesContext;
```

Comment:
This will also fail flow tests, and I don't know how to access the `PreferencesContext` here, other than this (I couldn't really figure out how to use the `Consumer` here, so I went with `ContextType`). Is there a better way?